torch/_dynamo/variables/builder.py

```diff
@@ -1273,7 +1273,9 @@ def wrap_tensor(self, value: torch.Tensor):
         ):
             unimplemented("torch.compile does not support strided NestedTensor")
 
-        if is_sparse_any(value):
+        # Reject sparse, but not coo.
+        # TODO: remove this altogether when non-coo sparsity propagation is ready
+        if is_sparse_any(value) and not value.is_sparse:
```

Comment:
If this is a torch.export only feature, gate it as such. If not, please add tests that show that guarding on sparse works (I have vague concerns).

Reply:
will follow up on this